recipes/bt/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "bt" %}
+{% set version = "0.2.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d162d71aaaaf7265a848d1fc0040f503add32dac2f9f3127bece0d74c22efb9b 
+
+build:
+  noarch: python
```

Comment:
I can't tell, is this pure Python or not? Cython and a C compiler are listed below, and the upstream repo `setup.py` mentions an extension module. But the repo itself only has Python code...

Reply:
yep sorry just copy-pasting from my previous repos